### PR TITLE
fix: Test failures

### PR DESCRIPTION
**Pull request by victor-yanev**
**Description**:

Updates the following packages:
- `@hashgraph/hedera-local` to ^2.19.1

Fixes the following tests:
- `Multicall.js` (brings up the call data size accordingly after the new limit from [this PR](https://github.com/hashgraph/hedera-mirror-node/pull/7796))
- `Vote.js` (reverts the admin of the `voteProxy` which is changed in one of the test scenarios)
- `ExampleGovernor.js` (adds missing `await tx.wait()` statement)
- `blindAuction.js` (optimizes tests with sleeps depending on the remaining time to the calculated `revealEnd` and `biddingEnd` timestamps)
- `pausable.js` (fixes race condition of transactions in test cases)
- `TypeOps.js` (fixes tests that are failing after solidity version update from `0.8.20` to `0.8.23` in https://github.com/hashgraph/hedera-smart-contracts/commit/b7224a1a7fd22725568146acf3ba9ec3d957ec76)
- Some other small fixes of unresolved variables and missing imports in other test files

Remaining tests that are failing:
- `diamond.js` - deploy of contract fails with `INVALID_ACCOUNT_ID` after solidity upgrade from `0.8.20` to `0.8.23`
- `erc-2612.js` - deploy of contract fails with `INVALID_ACCOUNT_ID`

**Related issue(s)**: https://github.com/hashgraph/hedera-smart-contracts/issues/685

Fixes https://github.com/hashgraph/hedera-smart-contracts/issues/685

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: test/hts-precompile/token-transfer/tokenTransferContract.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ const {
   pollForNewSignerBalanceUsingProvider,
 } = require('../../../utils/helpers');
 
-describe('TokenTransferContract Test Suite', function () {
+describe('TokenTransferContract Test Suite', async function () {
```
Comment (quiet-node) [inline test/hts-precompile/token-transfer/tokenTransferContract.js:30]: Why do we need `async` here?

File: test/openzeppelin/governor/ExampleGovernor.js [hunk 2]
```diff
@@ -107,14 +107,15 @@ describe('@OZGovernor Test Suite', function () {
...
-    await governor
+    const tx = await governor
       .connect(deployer)
       .execute(
         actions.targets,
         actions.values,
         actions.calldatas,
         descriptionHash
       );
+    await tx.wait();
```
Comment (quiet-node) [inline test/openzeppelin/governor/ExampleGovernor.js:118]: I like these additions it helps the mirror-node has more time to update records.

**Review by quiet-node — COMMENTED** [review]
Nice work overall just 1 question

**Comment by quiet-node** [comment]
Also a UT failed here https://github.com/hashgraph/hedera-smart-contracts/actions/runs/8377264254/job/22938948541?pr=707#step:10:276

**Review by georgi-l95 — DISMISSED** [review]
LG

File: test/hts-precompile/token-transfer/tokenTransferContract.js (hunk 1 above)
Comment (quiet-node) [reply test/hts-precompile/token-transfer/tokenTransferContract.js:30]: Hmm I see. I think we should leave the `async` keyword out and ignore the warning for now then could you please create a ticket to address this issue so we can come back and figure it out later?

File: test/multicall/Multicall.js [hunk 3]
```diff
@@ -266,8 +264,9 @@ describe('Multicall Test Suite', function () {
       expect(bytes).to.gte(42624);
     });
 
-    it('should NOT be able to aggregate 100 calls to processLongOutput', async function () {
-      const n = 100;
+    it('should NOT be able to aggregate 115 calls to processLongOutput', async function () {
+      const n = 115;
+      const maxDataSize = 25 * 1024 * 2; // 25 kb
```
Comment (quiet-node) [inline test/multicall/Multicall.js:269]: Q: this is 50KBs, not 25Kbs like the comment says right?

Comment (victor-yanev) [reply test/multicall/Multicall.js:269]:
```java
    private void validateContractData(final ContractCallRequest request) {
        var data = request.getData();
        if (data != null
                && !evmProperties.getDataValidatorPattern().matcher(data).find()) {
            throw new InvalidParametersException(
                    "data field of size %d contains invalid hexadecimal characters or exceeds %d characters"
                            .formatted(
                                    data.length(),
                                    evmProperties.getMaxDataSize().toBytes() * 2L));
        }
    }
```

Basically 2 hex characters represent 1 byte => maxDataSize is 25kb, which is 25 * 1024 * 2 in hexadecimal characters. Maybe the comment is a bit confusing, sorry.

Comment (quiet-node) [reply test/multicall/Multicall.js:269]: Ahh I see so it indicates the length of the hex string. It makes sense now! Thanks for the clarification!

**Review by quiet-node — APPROVED** [review]
LGTM! Thanks for the work!

**Pull request merged**